README.rst

```diff
@@ -770,16 +779,16 @@ to install using pip and a virtualenv::
 
     virtualenv env
     source env/bin/activate
-    python synapse/python_dependencies.py | xargs -n1 pip install
-    pip install setuptools_trial mock
+    python synapse/python_dependencies.py | xargs pip install
+    pip install lxml mock
```

Comment:
a python lib, or a C one? libxslt1-dev is already mentioned in "Installing from source".